nsqd/nsqd.go

```diff
@@ -550,7 +569,7 @@ func (n *NSQD) DeleteExistingTopic(topicName string) error {
 	return nil
 }
 
-func (n *NSQD) Notify(v interface{}) {
+func (n *NSQD) Notify(v notifyContext) {
```

Comment:
don't think this change is necessary, right?

Reply:
done.